.github/CONTRIBUTING.md

```diff
@@ -1 +1,52 @@
-We appreciate all contributions to improve MMDetection. Please refer to [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) in MMCV for more details about the contributing guideline.
```

Comment:
I think we can use something quite similar to upstream version.

-----
We appreciate all contributions to improve OTEDetection. Please refer to [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) in MMCV for more details about the contributing guideline.

Reply:
Done as you suggested.